docs/swagger/spec.yml

```diff
@@ -3463,3 +3567,123 @@ definitions:
         items:
           type: string
           format: uuid
+
+  AnnotationFeature:
+    type: object
+    properties:
+      type:
+        type: string
+      properties:
+        allOf:
+        - $ref: '#/definitions/BaseModel'
+        - $ref: '#/definitions/TimeModelMixin'
+        - $ref: '#/definitions/UserTrackingMixin'
+        - type: object
+        properties:
+          label:
+            type: string
+          description:
+            type: string
+          machineGenerated:
+            type: boolean
+          confidence:
+            type: number
+            format: float
+          qualityCheck:
+            type: string
+            enum:
+              - PASS
+              - FAIL
+              - NEUTRAL
+          projectId:
+            type: string
+            format: uuid
+      geometry:
+        $ref: "#/definitions/Geometry"
+
+  AnnotationFeatureUpdate:
+    type: object
+    properties:
+      type:
+        type: string
+      properties:
+        type: object
+        properties:
+          label:
+            type: string
+          description:
+            type: string
+          machineGenerated:
+            type: boolean
+          confidence:
+            type: number
+            format: float
+          qualityCheck:
+            type: string
+            enum:
+              - PASS
+              - FAIL
+              - NEUTRAL
+      geometry:
+        $ref: "#/definitions/Geometry"
+
+  AnnotationFeatureCreate:
+    type: object
+    properties:
+      type:
+        type: string
+      properties:
+        allOf:
+        - $ref: '#/definitions/BaseCreate'
+        - type: object
+        properties:
+          label:
+            type: string
+          description:
+            type: string
+          machineGenerated:
+            type: boolean
+          confidence:
+            type: number
+            format: float
+          qualityCheck:
+            type: string
+            enum:
+              - PASS
+              - FAIL
+              - NEUTRAL
+      geometry:
+        $ref: "#/definitions/Geometry"
+
+  AnnotationFeatureCollection:
+    type: object
+    properties:
+      type:
+        type: string
+      features:
+        type: array
+        items:
+          $ref: '#/definitions/AnnotationFeature'
+
+  AnnotationFeatureCollectionPaginated:
+    type: object
+    allOf:
+      - $ref: '#/definitions/PaginatedResponse'
```

Comment:
@lewfish -- looks like we might actually want to update the spec for information feeds.

@alkamin pointed out to me yesterday that a feature collection can have additional properties, which would allow pagination with geojson -- so I think your original proposal was probably better with the original pagination properties